org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NullTestCase.xtend

```diff
@@ -0,0 +1,60 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+
+class NullTestCase extends AbstractWollokInterpreterTestCase {
+	
+	@Test
+	def void messageSentToNull() {
+		'''
+		assert.throwsException({ null.sayHi() })
```

Comment:
Is it possible to assert that the correct exception is thrown?
(i.e. differentiate "does not understand" from "message to nothing".